app/controllers/api/v2/app_groups_controller.rb

```diff
@@ -166,6 +166,23 @@ def update_cost
       affected_app: affected_app
     }, status: :ok and return
   end
+  def delete
+    @helm_infrastructure = HelmInfrastructure.find_by(cluster_name: params[:id])
```

Comment:
do you sent cluster_name inside `params[:id]`? or should we change this to `params[:cluster_name]`?